ArithmeticTools/Rational.swift

```diff
@@ -24,9 +24,7 @@ public struct Fraction: Rational {
     /// Creates a `Fraction` value with a given `numerator` and `denominator`.
     public init(_ numerator: Int, _ denominator: Int) {
 
```

Comment:
Now that all this is stuff is a one-liner, I'd say we can crunch them all together with no whitespace.